tests/queries/0_stateless/02732_rename_after_processing.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# see 01658_read_file_to_stringcolumn.sh
+CLICKHOUSE_USER_FILES_PATH=$(clickhouse-client --query "select _path, _file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
+
+# Prepare data
+mkdir -p ${CLICKHOUSE_USER_FILES_PATH}
+echo '"id","str","int","text"' > ${CLICKHOUSE_USER_FILES_PATH}/tmp.csv
+echo '1,"abc",123,"abacaba"' >> ${CLICKHOUSE_USER_FILES_PATH}/tmp.csv
+echo '2,"def",456,"bacabaa"' >> ${CLICKHOUSE_USER_FILES_PATH}/tmp.csv
+echo '3,"story",78912,"acabaab"' >> ${CLICKHOUSE_USER_FILES_PATH}/tmp.csv
+echo '4,"history",21321321,"cabaaba"' >> ${CLICKHOUSE_USER_FILES_PATH}/tmp.csv
```

Comment:
Some other tests can use `user_files` folders in parallel. It's better to use uniq test name for these files. You can use `$CLICKHOUSE_TEST_UNIQUE_NAME` in filenames or create a folder with unique name for this test (and it will also allow to execute this test in parallel).